include/deal.II/meshworker/scratch_data.h

```diff
@@ -801,10 +813,228 @@ namespace MeshWorker
     /** @} */ // CurrentCellEvaluation
 
     /**
-     * Return a reference to the used mapping.
+     * @name Evaluation of jumps in finite element fields and their derivatives on the current interface
      */
-    const Mapping<dim, spacedim> &
-    get_mapping() const;
+    /** @{ */ // CurrentInterfaceJumpEvaluation
+
+    /**
+     * For the solution vector identified by @p global_vector_name, compute
+     * the jumps in the values of the function at the quadrature points, and
+     * return a vector with the correct type deduced by the interface Extractor
+     * you passed as the
+     * @p variable argument.
+     *
+     * Before you can call this method, you need to call the
+     * extract_local_dof_values() method at least once, passing the same
+     * @p global_vector_name string, and the same type for the variable @p dummy.
+     *
+     * If you have not previously called the extract_local_dof_values() method,
+     * this function will throw an exception.
+     *
+     * For this function to work properly, the underlying FEInterfaceValues
+     * object for which you called one of the reinit() functions, must have
+     * computed the information you are requesting. To do so, the update_values
+     * flag must be an element of the list of UpdateFlags that you passed to the
+     * constructor of this object. See "The interplay of UpdateFlags, Mapping,
+     * and FiniteElement" in the documentation of the FEValues class for more
+     * information.
+     */
+    template <typename Extractor, typename Number = double>
+    const std::vector<
+      typename FEInterfaceViews::View<dim, spacedim, Extractor>::
+        template solution_value_type<Number>> &
+    get_jumps_in_values(const std::string &global_vector_name,
+                        const Extractor &  variable,
+                        const Number       dummy = Number(0));
```

Comment:
This is a copy-paste-adjust of https://github.com/dealii/dealii/blob/master/include/deal.II/meshworker/scratch_data.h#L576-L602. I use the plural `jumps` in `get_jumps_in_values()` as it returns a vector with each entry being a jump in values across the interface. In the accompanying text, the changes are:
- `compute the values of the function` -> `compute the jumps in the values of the function`
- `For this function to work properly, the underlying FEValues, FEFaceValues, or FESubfaceValues object for which...` -> `For this function to work properly, the underlying FEInterfaceValues object for which...`

I would have applied the same reasoning and changes to the other methods.